ansible/checker.yml

```diff
@@ -0,0 +1,46 @@
+# This playbook deploys a simple stand alone checker for SAP/HANS.
+
+- hosts: all
+
+  tasks:
+    - name: include load_facts
+      import_role:
+        name: load_facts
+      tags:
+        - always
+
+    - name: include 1.1.1 remediate
+      import_role:
+        name: '1.1.1'
+      tags:
+        - 1.1.1
+
+    - name: include 1.1.1.runtime remediate
+      import_role:
+        name: '1.1.1.runtime'
+      tags:
+        - 1.1.1.runtime
+
+    - name: include 1.2.1 remediate
+      import_role:
+        name: '1.2.1'
+      tags:
+        - 1.2.1
+
+    - name: include 1.3.5 remediate
+      import_role:
+        name: '1.3.5'
+      tags:
+        - 1.3.5
+
+    - name: set_global_results
+      set_fact:
+        global_results: "{{ hostvars.hana1.tests }}{{ hostvars.hana2.tests }}"
+      run_once: yes
```

Comment:
I know that this is not right and I am still working on it.  It is an interaction issue ara_record block below. That block must be done on local host, and should be done once to write out all test results, but test results are stored as per host fact.
So, the need to loop over all hosts in inventory merge all test results. Still working on that, but wanted to get the rest out for review.

Reply:
Should be OK now.